pkg/infinity/client.go

```diff
@@ -202,26 +202,28 @@ func (client *Client) req(ctx context.Context, url string, body io.Reader, setti
 		logger.Error("url is not in the allowed list. make sure to match the base URL with the settings", "url", req.URL.String())
 		return nil, http.StatusUnauthorized, 0, errorsource.DownstreamError(errors.New("requested URL is not allowed. To allow this URL, update the datasource config Security -> Allowed Hosts section"), false)
 	}
-	logger.Debug("yesoreyeram-infinity-datasource plugin is now requesting URL", "url", req.URL.String())
+	logger.Debug("yesoreyeram-infinity-datasource plugin requesting URL", "host", req.URL.Hostname(), "url_path", req.URL.Path, "method", req.Method, "type", query.Type)
```

Comment:
As we are changing this - we will have plugin id as log label, do you think we also need it in message. I'd update to just 

```suggestion
	logger.Debug("requesting URL", "host", req.URL.Hostname(), "url_path", req.URL.Path, "method", req.Method, "type", query.Type)
```